README.md

```diff
@@ -22,59 +22,78 @@ git clone --recursive https://github.com/NationalGenomicsInfrastructure/genomics
+sudo ln -s /opt/homebrew/opt/pango/lib/libpango-1.0.dylib /usr/local/lib/pango-1.0
+sudo ln -s /opt/homebrew/opt/harfbuzz/lib/libharfbuzz.dylib /usr/local/lib/harfbuzz
+sudo ln -s /opt/homebrew/opt/fontconfig/lib/libfontconfig.1.dylib /usr/local/lib/fontconfig-1
+sudo ln -s /opt/homebrew/opt/pango/lib/libpangoft2-1.0.dylib /usr/local/lib/pangoft2-1.0
 ```
 
 </details>
 
-Install the dependencies and the package (the `pip install -r requirements_dev.txt` can be skipped on a production server)
+
+#### Install via `conda` [recommended]
+
+If you are using `conda`, we recommend creating a new environment using `conda-lock` to ensure that the environment is reproducible across different systems.
+
+**Create the environment using `conda-lock` [recommended]**
+
+To create a new environment using `conda-lock`, first install `conda-lock`, if you haven't already, using one of the [available methods](https://github.com/conda/conda-lock/tree/main?tab=readme-ov-file#installation). 
+
+Then, run the following command in the root directory of the repository:
 
 ```bash
-pip install -r requirements.txt
-pip install -r requirements_dev.txt
-python setup.py install
+conda-lock install --name <env_name>
 ```
 
-<details>
+This will create a new environment with the name `<env_name>` and install all the required dependencies.
 
-<summary>Note on MacOS installations</summary>
+<details>
 
-### Fix library issues
+<summary>Using default conda [alternative]</summary>
 
-If you are using MacOS, you might run into some issues with the libraries when running the app. 
+#### Create the environment using `conda`
 
-In case you get the `OSError: cannot load library 'gobject-2.0-0'` error (or similar), install `glib` and `pango` via [Homebrew](https://brew.sh/):
+If you prefer to create the environment using `conda` directly, you can use the provided `conda_requirements.yml` file.
 
 ```bash
-brew install glib
-brew install pango
+conda env create -n <env_name> -f conda_requirements.yml
 ```
 
-and then symlink the libraries to `/usr/local/lib`:
+where `<env_name>` is the name of the environment you want to create.  Then, activate the environment with:
 
 ```bash
-sudo ln -s /opt/homebrew/opt/glib/lib/libgobject-2.0.0.dylib /usr/local/lib/gobject-2.0
-sudo ln -s /opt/homebrew/opt/pango/lib/libpango-1.0.dylib /usr/local/lib/pango-1.0
-sudo ln -s /opt/homebrew/opt/harfbuzz/lib/libharfbuzz.dylib /usr/local/lib/harfbuzz
-sudo ln -s /opt/homebrew/opt/fontconfig/lib/libfontconfig.1.dylib /usr/local/lib/fontconfig-1
-sudo ln -s /opt/homebrew/opt/pango/lib/libpangoft2-1.0.dylib /usr/local/lib/pangoft2-1.0
+conda activate <env_name>
 ```
 
 </details>
 
+#### Install via `pip` [alternative]
+
+Install the dependencies and the package (the `pip install -r requirements_dev.txt` can be skipped on a production server)
+
+```bash
+pip install -r requirements.txt
+pip install -r requirements_dev.txt
+python setup.py install
+```
+
```

Comment:
I'm not sure if this is enough anymore. We install pandas through conda, in addition to the packages needed for Agreements and Invoicing.